Assets/_Scripts/2Models/Tile.cs

```diff
@@ -49,6 +49,11 @@ public override string ToString()
         {
             return $"Tile: {X}, {Y}";
         }
-    }
 
+        public override bool Equals(object obj)
```

Comment:
Make this an operator method

Reply:
public static Calculator operator + (Calculator Calc1, Calculator Calc2)

Reply:
Done

Reply:
Nvm, I don't know why, but adding those operator overrides makes the test fail.

Reply:
I found a fix and pushed it. Still a bug that makes the TestAdjacentTilesBorders run red, will investigate later.